### PR TITLE
Allow to set the hostname for routes to prometheus and alertmanager

### PR DESCRIPTION
**Pull request by dabelenda**

### CODE REVIEW

File: roles/openshift_prometheus/README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ For default values, see [`defaults/main.yaml`](defaults/main.yaml).
 
 - `openshift_prometheus_<COMPONENT>_image_version`: specify image version for the component 
 
+- `openshift_prometheus_<COMPONENT>_hostname`: specify the hostname for the route of that compontent: defaults to `<COMPONENT>-{{openshift_prometheus_namespace}}.{{openshift_master_default_subdomain}}`
+
```
Comment (zgalor) [inline roles/openshift_prometheus/README.md]: Would change the documentation to explicit `openshift_prometheus_hostname` and `openshift_prometheus_alerts_hostname`.
There are only 2 routes, and <COMPONENT> usually refers to: prometheus/alertmanager/alertbuffer/oauth-proxy

File: roles/openshift_prometheus/defaults/main.yaml [hunk 2]
```diff
@@ -4,6 +4,10 @@ openshift_prometheus_state: present
 
 openshift_prometheus_namespace: openshift-metrics
 
+# defaults hosts for routes
+openshift_prometheus_prometheus_hostname: prometheus-{{openshift_prometheus_namespace}}.{{openshift_master_default_subdomain}}
```
Comment (zgalor) [inline roles/openshift_prometheus/defaults/main.yaml]: I would change this to `openshift_prometheus_hostname`. This better aligns with other variable pattern

File: roles/openshift_prometheus/README.md [hunk 3]
```diff
@@ -23,6 +23,10 @@ For default values, see [`defaults/main.yaml`](defaults/main.yaml).
 
 - `openshift_prometheus_<COMPONENT>_image_version`: specify image version for the component 
 
+- `openshift_prometheus_hostname`: specify the hostname for the route to prometheus `prometheus-{{openshift_prometheus_namespace}}.{{openshift_master_default_subdomain}}`
+
+- `openshift_prometheus_alerts_hostname`: specify the hostname for the route to prometheus `prometheus_alerts-{{openshift_prometheus_namespace}}.{{openshift_master_default_subdomain}}`
```
Comment (zgalor) [inline roles/openshift_prometheus/README.md]: 'specify the hostname for the route to prometheus-alerts'

Comment (dabelenda) [reply roles/openshift_prometheus/README.md]: true... copy-pasted too fast...

File: roles/openshift_hosted/tasks/create_groups.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (michaelgugino) [inline roles/openshift_hosted/tasks/create_groups.yml]: What is going on with this file?  I think you accidentally commit some local testing changes.

**Comment by dabelenda** [comment]
Oh sorry about that... I did not see this file...

**Comment by dabelenda** [comment]
rebase was needed so I have done the rebase.

**Comment by dabelenda** [comment]
/test logging
/test install
Considering the tests fail on logging/fluentd I guess my change is not responsible.

**Pull request merged**